src/bin/uhyve.rs

```diff
@@ -186,17 +179,14 @@ pub struct MemoryArgs {
 #[derive(Clone, Debug, Default, Deserialize, Merge, Parser)]
 #[cfg_attr(test, derive(PartialEq))]
 #[serde(default)]
+#[merge(strategy = merge::option::overwrite_none)]
```

Comment:
I believe this is fine – I did not apply the strategy for the entire structure because I initially missed that all fields had the same strategy.